docs/adding-a-new-ballerina-module.md

```diff
@@ -44,15 +44,15 @@ This is a step-by-step guide on creating a new Ballerina module. It will guide y
         * 6.11.8 [The `settings.gradle` file](#the-settingsgradle-file-required)
         * 6.11.9 [The `spotbugs-exclude.xml` file](#the-spotbugs-excludexml-file-optional)
 7. [Add the new module](#add-the-new-module)
-    * 7.1 [Add the module to the Ballerina Standard Library](#add-the-module-to-the-ballerina-standard-library-optional)
+    * 7.1 [Add the module to the Ballerina library](#add-the-module-to-the-ballerina-standard-library-optional)
```

Comment:
Anchor will break here. Have to update that as well.

Reply:
Updated.